stl/inc/spanstream

```diff
@@ -64,7 +64,7 @@ public:
     // N4892 [spanbuf.assign], assignment and swap
     basic_spanbuf& operator=(const basic_spanbuf&) = delete;
 
-    basic_spanbuf& operator=(basic_spanbuf&& _Right) {
+    basic_spanbuf& operator=(basic_spanbuf&& _Right) noexcept {

```

Comment:
Perhaps we should refactor this move assignment operator to avoid move construction.

Reply:
I'm working on this in my (another) PR.